domains/certificates/Query.API/API/Query.API/Controllers/SignUpsController.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading;
+using System.Threading.Tasks;
+using API.CertificateGenerationSignUpService;
+using API.Query.API.ApiModels.Requests;
+using API.Query.API.ApiModels.Responses;
+using FluentValidation;
+using FluentValidation.AspNetCore;
+using Marten;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using static API.CertificateGenerationSignUpService.CreateSignUpResult;
+
+namespace API.Query.API.Controllers;
+
+[Authorize]
+[ApiController]
+public class SignUpsController : ControllerBase
+{
+    /// <summary>
+    /// Signs up a metering point for granular certificate generation
```

Comment:
Are these summaries used in the opanapi doc?

Reply:
Yes